src/predict_request_validation_utils.cpp

```diff
@@ -864,6 +976,21 @@ Status RequestValidator<RequestType, InputTensorType, IteratorType, ShapeType>::
             continue;
         }
 
+        if (checkIfStringFormatUsed(proto, name, inputInfo)) {
```

Comment:
In deserialization the order is to first check if this is string processing **then** if this is nativefile format processing. We should have the same logic in validation